apache-spark/spark3.0/image-classification/src/main/scala/com/examples/ImageClassificationExample.scala

```diff
@@ -23,22 +21,61 @@ import org.apache.spark.sql.SparkSession
 object ImageClassificationExample {
 
   def main(args: Array[String]): Unit = {
+    var outputPath: String = null
+    if (args.length > 0) {
+      outputPath = args(0)
+    } else {
+      println("S3 output location not specified, printing results to output stream")
+    }
+
     val spark = SparkSession.builder()
-      .master("local[*]")
-      .appName("Image Classification")
+      .appName("ImageClassificationExample")
       .getOrCreate()
+    val sc = spark.sparkContext
+    sc.setLogLevel("ERROR")
+
+    var df = spark.read.format("image").option("dropInvalid", true).load("s3://alpha-djl-demos/temp/images/")
+    df.printSchema()
+    // root
+    // |-- image: struct (nullable = true)
+    // |    |-- origin: string (nullable = true)
+    // |    |-- height: integer (nullable = true)
+    // |    |-- width: integer (nullable = true)
+    // |    |-- nChannels: integer (nullable = true)
+    // |    |-- mode: integer (nullable = true)
+    // |    |-- data: binary (nullable = true)
 
-    val df = spark.read.format("image").option("dropInvalid", true).load("s3://alpha-djl-demos/temp/images")
-    df.select("image.origin", "image.width", "image.height").show(truncate = false)
+    df = df.filter("image.nChannels=3") // The model expects RGB images
+    df.select("image.origin", "image.width", "image.height", "image.nChannels").show(truncate = false)
```

Comment:
We don't need nChannels, right? We already filtered on nChannels

Reply:
Just to show the user it is indeed nChannels=3.